resources/todomvc/architecture-examples/vue/src/components/TodosComponent.vue

```diff
@@ -7,24 +7,17 @@
                 <label class="toggle-all-label" htmlFor="toggle-all-input"> Toggle All Input </label>
             </div>
             <ul class="todo-list">
-                <li class="todo" :key="todo.id" v-for="todo in filteredTodos" :class="{ completed: todo.completed, editing: todo === editing }">
-                    <div class="view">
-                        <input type="checkbox" v-model="todo.completed" class="toggle" />
-                        <label @dblclick="startEdit(todo)">{{ todo.title }}</label>
-                        <button class="destroy" @click.prevent="deleteTodo(todo)"></button>
-                    </div>
-                    <input type="text" class="edit" v-model="todo.title" @keyup.enter="finishEdit" @blur="finishEdit" v-todoFocus="todo === editing" />
-                </li>
+                <TodoItem v-for="todo in filteredTodos" :key="todo.id" v-memo="[todo.completed, todo.title]" :todo="todo" @delete-todo="deleteTodo" @edit-todo="editTodo" @toggle-todo="toggleTodo" />
```

Comment:
It would be good to see the numbers without `v-memo`. My understanding is that it is superfluous. Indeed according to https://vuejs.org/guide/best-practices/performance.html#props-stability Vue memoizes the props already, so if `todo` stays the same (and I believe it changes only when we edit a property thanks to the internal use of proxies) the update will be skipped already.
Therefore I believe that using `v-memo` adds work for no benefit. It would be good to check :-)

Reply:
@julienw - let me remove the v-memo call. There wasn't any measurable difference in using it and I was going to remove it as well.

Reply:
done